controllers/verification_test.go

```diff
@@ -175,19 +168,6 @@ func (suite *VerificationTestSuite) TestVerifyInitUnsupportedEmail() {
 	resp = util.ExecuteTestRequest(util.CreateJSONTestRequest("/v2/verify/init", body), suite.router)
 	suite.Equal(http.StatusBadRequest, resp.Code)
 	util.AssertErrorResponseCode(suite.T(), resp, util.ErrEmailDomainNotSupported.Code)
-
-	// Test North Korea TLD for accounts registration
-	body.Intent = "registration"
-	body.Service = "accounts"
-	body.Email = "test@example.kp"
-	resp = util.ExecuteTestRequest(util.CreateJSONTestRequest("/v2/verify/init", body), suite.router)
-	suite.Equal(http.StatusBadRequest, resp.Code)
-	util.AssertErrorResponseCode(suite.T(), resp, util.ErrEmailDomainNotSupported.Code)
-
-	// Email with 'strict' TLD should be allowed for registration
```

Comment:
Were these test cases moved somewhere else?

Reply:
yes, they're in the accounts controller tests now